assets/data/effects/drauven/drauven_pc_upgraded_bloodedFerocity.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
To keep this and the parent files adjacent in the folder view, I would recommend naming it `drauven_pc_bloodedFerocity_upgraded`